docs/pages/enroll-resources/database-access/enroll-self-hosted-databases/sql-server-ad-pkinit.mdx

```diff
@@ -170,6 +177,12 @@ Teleport CRL to your Active Directory domain (using the path to the exported
 certutil -dspublish -f <PathToCRLFile.crl> TeleportDB
 ```
 
+If you use HSM run commands printed at the end of the CRL export. They will in form of:
+
+```powershell
+certutil -dspublish -f <PathToCRLFile.crl> TeleportDB <CRLCommonName>
```

Comment:
I'm not sure I would know what to put for the common name if I were reading these docs.

Should we instead say "run the commands that were printed by tctl?